src/main/org/deidentifier/arx/metric/MetricDefault.java

```diff
@@ -1,77 +1,77 @@
-/*
```

Comment:
Not sure why the review identifies some classes as completely replaced.  No wholesale cut and paste on my part, though I did rely on automated refactoring.